### PR TITLE
Added custom alc269vc for Samsung NP530U3C-A0F

### PR DESCRIPTION
**Pull request by BblDE3HAP**

### CODE REVIEW

**Comment by vit9696** [comment]
Thanks...

File: Resources/ALC269/Info.plist [hunk 1]
```diff
@@ -142,6 +142,14 @@
 				<key>Path</key>
 				<string>layout32.xml.zlib</string>
 			</dict>
+			<dict>
+				<key>Id</key>
+				<integer>33</integer>
+				<key>Comment</key>
+				<string>Custom ALC269VC for Samsung NP530U3C-A0F by BblDE3HAP</string>
+				<key>Path</key>
+				<string>Platforms33.xml.zlib</string>
```
Comment (BblDE3HAP) [inline Resources/ALC269/Info.plist]: My mistake, should be layout33.xml.zlib on 151

Comment (vit9696) [inline Resources/ALC269/Info.plist]: Ouch, bad idea to review code in such early mornings. Committed a fix :)

**Comment by BblDE3HAP** [comment]
Thank you, not me. (^_^)

Forgot to mention:
- just now I've realised that there is a mistake in 'Resources/ALC269/Info.plist' on line #151
- probably it can use platforms28, but actually I didn't check that;
- EAPD codec commander needed, otherwise after sleep no sound in int_speakers.

**Comment by BblDE3HAP** [comment]
Bad idea to push something at early morning, my bad, sorry...

**Comment by vit9696** [comment]
> probably it can use platforms28, but actually I didn't check that;

As for this it might be a good idea to check, the hashes are different, so I am not sure it is perfectly safe to.

**Comment by BblDE3HAP** [comment]
Well, thats what differs.
https://github.com/BblDE3HAP/test/compare/res_33?expand=1

**Comment by vit9696** [comment]
@BblDE3HAP I guess it is a good idea to check if it works ok in this case :)
cc @vandroiy2013

**Pull request merged**